crates/bws/src/main.rs

```diff
@@ -246,7 +246,7 @@ async fn process_commands() -> Result<()> {
 
     let Some(command) = cli.command else {
         let mut cmd = Cli::command();
-        cmd.print_help()?;
+        eprintln!("{}", cmd.render_help());
```

Comment:
```suggestion
        eprintln!("{}", cmd.render_help().ansi());
```

This technically breaks the styling that the clap help does, calling `.ansi()` will return the formatting. (We might want to do it only when color is enabled though)

Top is the current master, bottom is this PR without the ansi change:
![image](https://github.com/bitwarden/sdk/assets/725423/6e7f50e4-60c3-4d4d-b143-a6e49d78d221)

Printing it like this loses the underscores and the bold headers.

Reply:
Good catch, fixed in: 5b08cfd